### PR TITLE
added calendar shortcuts

### LINKED ISSUE #802: Add Calendar Keyboard Shortcuts
- When the user presses 'w' or 'm' it should switch to week and month view respectively

### PR DESCRIPTION
**Pull request by seanquiambao**
tried using a ternary operator, but i couldn't figure out how to handle default case since pressing any other key will change to the initial value of view.

used keycode to take into account of capital letters.

### CODE REVIEW

File: src/components/dynamic/admin/services/calendar/Events.jsx [hunk 1]
```diff
@@ -17,6 +17,19 @@ const CalendarEvents = () => {
   const [date, setDate] = useState(new Date());
   const [view, setView] = useState("month");
 
+  const handleShortcuts = (e) => {
+    switch (e.keyCode) {
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Events.jsx]: there is there a way to match it based on character like if e.character === "w" so we dont need to know the ASCII codes

Comment (seanquiambao) [reply src/components/dynamic/admin/services/calendar/Events.jsx:21]: so would i use `e.code === "KeyW"` or `e.key === "w" || e.key === "W"`

Comment (seanquiambao) [reply src/components/dynamic/admin/services/calendar/Events.jsx]: `e.code === "KeyW"` or just `e.key === "w" || e.key === "W"`?

File: src/components/dynamic/admin/services/calendar/Events.jsx [hunk 2]
```diff
@@ -17,6 +17,19 @@ const CalendarEvents = () => {
...
+  const handleShortcuts = (e) => {
+    switch (e.key) {
+      case "m":
+        setView("month");
+        break;
+      case "w":
+        setView("week");
+        break;
+      default:
+        break;
```
Comment (shahdivyank) [inline src/components/dynamic/admin/services/calendar/Events.jsx]:
```suggestion
```

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**